src/AuthState.ts

```diff
@@ -29,9 +38,31 @@ export interface AuthState {
     /**
      * Was there a signin or silent renew error?
      */
-    error?: Error;
+    error?: ErrorContext;
 }
 
+/**
+ * Represents an error while execution of a signing, renew, ...
+ *
+ * @public
+ */
+export type ErrorContext = Error & {
+    cause?: unknown;
```

Comment:
to avoid confusion we might call `cause` better `innerError`, which is what others are doing. I guess the type is for this good enough, as its now the original error coming from the underlying library.

Reply:
Sure. Done.